### DISCUSSION TITLE
GitHub Universe 2022 - fund, accelerators and sponsorships.

### DISCUSSION THREAD

**Discussion by a0m0rajab** [community/community#38712] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

If you attended GitHub Universe today, you would have heard about the new accelerators and funding programs, how you think that this programs will empower new open source developers and the whole ecosystem?

**Comment by ossanna16** [comment]
Hi there @a0m0rajab and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
